.github/workflows/tests-oracle.yml

```diff
@@ -0,0 +1,71 @@
+name: Oracle Tests
+
+on:
+  push:
+    paths:
+      - 'src/test/java/org/jabref/model/database/**'
+      - 'src/test/java/org/jabref/logic/shared/**'
+      - '.github/workflows/tests-oracle.yml'
+      - 'build.gradle'
+  schedule:
+    # run on each Wednesday
+    - cron: '2 3 * * 3'
+
+jobs:
+  oracletests:
+    name: Oracle tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v1
+        with:
+          depth: 1
+          submodules: false
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 13
+      - uses: actions/cache@v1
+        name: Restore gradle chache
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: |
+            ${{ runner.OS }}-gradle-${{ env.cache-name }}-
+            ${{ runner.OS }}-gradle-
+            ${{ runner.OS }}-
+      - uses: actions/cache@v1
+        name: Restore gradle wrapper
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+      - name: Cache Oracle XE downloads
```

Comment:
> [warning]Cache size of ~2455 MB (2573776080 B) is over the 400MB limit, not saving cache.

I think github will not increase the cache size that dramatically in the near future. Thus, this cache can be removed (which reduces the build time by 2 mins - not much but better than nothing).

Reply:
For caching, see the discussion here: https://github.com/actions/cache/issues/6

Reply:
Result of the discussion:

> There may be another update soon to up the limit a little (~700MB or so), [...] We would still have the 2GB per repo limit until we integrate billing, so there would be some concern about thrashing the cache if the individual limit matched the per repo limit.